src/test/java/org/zendesk/client/v2/RealSmokeTest.java

```diff
@@ -1,5 +1,31 @@
 package org.zendesk.client.v2;
 
+import static org.hamcrest.CoreMatchers.anyOf;
```

Comment:
@ajarnold87 it might be better to not reorganise the imports. You can see if your IDE is correctly configured here:

https://github.com/cloudbees/zendesk-java-client/blob/master/CONTRIBUTING.adoc#ide-configuration